### PR TITLE
CSL style for the journal Early Christianity (EC)

### PR DESCRIPTION
**Pull request by carljohanberglund**

### CODE REVIEW

File: early-christianity.csl [hunk 1]
```diff
@@ -0,0 +1,792 @@
...
+    </author>
+    <category citation-format="note"/>
+    <category field="theology"/>
+    <issn>1868-7032</issn>
+    <issn>1868-8020</issn>
+    <summary>Early Christianity</summary>
+    <updated>2015-08-25T09:08:29+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="sv-SE">
```
Comment (adam3smith) [inline early-christianity.csl]: can you submit in Swedish? I thought it was just English in German. If I'm right about this, please remove this locale (down to line 32).

**Comment by adam3smith** [comment]
that's my only concern, everything else looks great.

File: early-christianity.csl [hunk 2]
```diff
@@ -0,0 +1,792 @@
...
+    <link href="http://www.zotero.org/styles/society-of-biblical-literature-fullnote-bibliography" rel="template"/>
+    <link href="https://www.mohr.de/en/journals/early-christianity-ec#menu4" rel="documentation"/>
+    <author>
+      <name>Carl Johan Berglund</name>
+      <email>cajo@cajo.nu</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="theology"/>
+    <issn>1868-7032</issn>
+    <issn>1868-8020</issn>
```
Comment (rmzelle) [inline early-christianity.csl]: It would also be better to change this to `<eissn>1868-8020</eissn>`

**Comment by carljohanberglund** [comment]
Done and done.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**